pkg/bgp/speaker/events.go

```diff
@@ -230,7 +231,7 @@ func (s *MetalLBSpeaker) handleNodeEvent(k nodeEvent) types.SyncState {
 	if s.announcePodCIDR {
 		l.Debug("announcing pod CIDR(s)")
 		if err := s.announcePodCIDRs(*k.podCIDRs); err != nil {
-			l.WithError(err).WithField("CIDRs", k.podCIDRs).Error("Failed to announce pod CIDRs")
+			l.WithError(err).WithField(logfields.CIDRS, k.podCIDRs).Error("Failed to announce pod CIDRs")
```

Comment:
same

Reply:
thanks for review.
you are right.
Update it.